sympy/stats/crv_types.py

```diff
@@ -2320,6 +2320,32 @@ def pdf(self, x):
     def sample(self):
         return random.normalvariate(self.mean, self.std)
 
+    def _sample_numpy(self, **kwargs):
+        mean, std = self.mean, self.std
+        if isinstance(mean, Symbol) or isinstance(std, Symbol):
```

Comment:
Just a suggestion for the future work, use, something like, `mean.has(Symbol) or std.has(Symbol)`. No need to make changes now, as this is a prototype.